test/unit/three_scale/invitation_email_delivery_observer_test.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ThreeScale::InvitationEmailDeliveryObserverTest < ActiveSupport::TestCase
+  include ActionMailer::TestHelper
+
+  def setup
+    InvitationMailer.register_observer ThreeScale::InvitationEmailDeliveryObserver
+  end
+
+  def test_update_date_after_sent_only
+    @mailer = nil
```

Comment:
why instance variables if these are apparently used only inside this test method?